services/can_logger/docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+version: '3.1'
+services:
+  can-logger:
+    build:
+      context: ../can_logger
+    restart: always
+    environment:
+      db_user: "avena"
+      db_host: "postgres"
+      db_database: "avena"
+      db_password: "password"
+      db_port: "5432"
+      host_ip: "172.17.0.1"
```

Comment:
From `can_logger.py`:
```
host_ip = os.environ['socketcand_ip']
host_port = os.environ['socketcand_port']
host_interface = os.environ['can_interface']
logging = os.environ['log']
```

I think you mixed up the names somewhere? IE It should be `socketcand_ip: "xx.yy.zz.aa"` here or `host_ip = os.environ['host_ip']` there